.travis.yml

```diff
@@ -2,14 +2,12 @@ language: java
 
 sudo: false
 
-script:
-  - jdk_switcher use oraclejdk7
-  - mvn clean verify -P jdk7
-  - jdk_switcher use oraclejdk8
-  - mvn clean verify -P jdk8
+jdk:
+  - oraclejdk8
+  - oraclejdk7
 
-after_success:
-  - mvn clean test jacoco:report coveralls:report
+script:
```

Comment:
> Using after_success we'd get a successful build with a separate report on the pr regarding coverage.

I see three different checks on the PR, one for the build of the branch, one for the merge (?) and one for the coverage.